api/src/test/java/org/openmrs/test/jupiter/BaseContextSensitiveTest.java

```diff
@@ -34,6 +34,7 @@
 import java.util.Timer;
 import java.util.TimerTask;
 
+import org.apache.commons.io.FileUtils;
```

Comment:
we do not need FileUtils as we let JUnit create and delete the tempappdir

Reply:
@teleivo When I remove `import org.apache.commons.io.FileUtils;` I get this error https://pastebin.com/TiKe5rPs. Is it okay for me to conclude we still need the  `FileUtils` to provide operation based around the JDK File class?

Reply:
have you removed the code/lines I suggested? You introduced the import with your changes that I believe need to be removed as well. JUnit 5 TempDir creates and deletes the tempappdir